src/kubernetes_configmap_jwt_scan/test/conftest.py

```diff
@@ -0,0 +1,15 @@
+import sys
```

Comment:
This file is necessary when writing tests for this module. Otherwise, it errors out with: `argparse.ArgumentError: argument artsy_env: invalid choice...`

Reply:
I decided to keep the file after moving the test to `src/lib/test/test_k8s_configmap_jwt_scan.py`, but without tests in `src/kubernetes_configmap_jwt_scan/test/test_scan.py` it's not necessary.